pkg/status/combinedstatus-resolution.go

```diff
@@ -586,22 +617,22 @@ func handleCombinedFields(scName string, scData *statusCollectorData) *v1alpha1.
 	if len(scData.GroupBy) > 0 {
```

Comment:
FYI, I would find it easier to follow the code if the comment on the `statusCollectorData` said that each one is specific to a (Binding, StatusCollector, workload object) combination.